app/models/ability.rb

```diff
@@ -50,8 +50,10 @@ def not_signed_in
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can :show, User
     unless CONFIG['authentication']['ichain']['enabled']
-      can [:show, :create], Registration do |registration|
-        registration.new_record?
+      can :show, Registration
+
```

Comment:
I have restored the check for `new` and `create` method, however I don't understand why should we checking `new_record?` for `show` method? I think it just shouldn't exist here as we can't show registration unless user is registered and signed in.